Projects/UOContent/Mobiles/Townfolk/Banker.cs

```diff
@@ -82,7 +84,9 @@ public static int GetBalance(Mobile m, out List<Item> gold, out List<Item> check
 
         if (bank != null)
         {
-            foreach (var g in bank.FindItemsByType<Gold>())
+            gold = new List<Gold>();
+
+            foreach (var g in bank.FindItemsByType<Gold>(true))
```

Comment:
The argument is true by default, so this shouldn't "fix" the recurse issue?

Reply:
Removed the explicit flag since there are no differences. 
From latest test the recursion is working